script.js

```diff
@@ -0,0 +1,78 @@
+class Numberdrome {
+
+    constructor() {
+        this.numbers = [];
+    }
+
+
+    addNumber(n) {
+        this.numbers.push(n);
+    }
+
+    removeNumber(n) {
+        this.numbers.splice(this.numbers.indexOf(n), 1);
+    }
+
+    sum(){
+        return this.numbers.length > 0 ? this.numbers.reduce((a,b) => a + b, 0) : 0;
```

Comment:
unneeded ternary, reduce will return 0 on the empty array
add space after the comma